ForceTouchPlayer/Songs/Greensleeves.swift

```diff
@@ -0,0 +1,70 @@
+      (OVERRIDE_NOTE_AS4,DURATION_OVERRIDE_4), (OVERRIDE_NOTE_C5,DURATION_OVERRIDE_8), (OVERRIDE_NOTE_D5,DURATION_OVERRIDE_MINUS_8), (OVERRIDE_NOTE_DS5,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_D5,DURATION_OVERRIDE_8),
+      (OVERRIDE_NOTE_C5,DURATION_OVERRIDE_4), (OVERRIDE_NOTE_A4,DURATION_OVERRIDE_8), (OVERRIDE_NOTE_F4,DURATION_OVERRIDE_MINUS_8), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_A4,DURATION_OVERRIDE_8),
+      (OVERRIDE_NOTE_AS4,DURATION_OVERRIDE_4), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_8), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_MINUS_8), (OVERRIDE_NOTE_FS4,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_8),
+      (OVERRIDE_NOTE_A4,DURATION_OVERRIDE_4), (OVERRIDE_NOTE_FS4,DURATION_OVERRIDE_8), (OVERRIDE_NOTE_D4,DURATION_OVERRIDE_4), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_8),
+
+      (OVERRIDE_NOTE_AS4,DURATION_OVERRIDE_4), (OVERRIDE_NOTE_C5,DURATION_OVERRIDE_8), (OVERRIDE_NOTE_D5,DURATION_OVERRIDE_MINUS_8), (OVERRIDE_NOTE_DS5,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_D5,DURATION_OVERRIDE_8),
+      (OVERRIDE_NOTE_C5,DURATION_OVERRIDE_4), (OVERRIDE_NOTE_A4,DURATION_OVERRIDE_8), (OVERRIDE_NOTE_F4,DURATION_OVERRIDE_MINUS_8), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_A4,DURATION_OVERRIDE_8),
+      (OVERRIDE_NOTE_AS4,DURATION_OVERRIDE_MINUS_8), (OVERRIDE_NOTE_A4,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_8), (OVERRIDE_NOTE_FS4,DURATION_OVERRIDE_MINUS_8), (OVERRIDE_NOTE_E4,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_FS4,DURATION_OVERRIDE_8),
+      (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_MINUS_2),
+      (OVERRIDE_NOTE_F5,DURATION_OVERRIDE_2), (OVERRIDE_NOTE_E5,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_D5,DURATION_OVERRIDE_8),
+      
+      (OVERRIDE_NOTE_C5,DURATION_OVERRIDE_4), (OVERRIDE_NOTE_A4,DURATION_OVERRIDE_8), (OVERRIDE_NOTE_F4,DURATION_OVERRIDE_MINUS_8), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_A4,DURATION_OVERRIDE_8),
+      (OVERRIDE_NOTE_AS4,DURATION_OVERRIDE_4), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_8), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_MINUS_8), (OVERRIDE_NOTE_FS4,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_8),
+      (OVERRIDE_NOTE_A4,DURATION_OVERRIDE_4), (OVERRIDE_NOTE_FS4,DURATION_OVERRIDE_8), (OVERRIDE_NOTE_D4,DURATION_OVERRIDE_4),
+      (OVERRIDE_NOTE_F5,DURATION_OVERRIDE_2), (OVERRIDE_NOTE_E5,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_D5,DURATION_OVERRIDE_8),
+      (OVERRIDE_NOTE_C5,DURATION_OVERRIDE_4), (OVERRIDE_NOTE_A4,DURATION_OVERRIDE_8), (OVERRIDE_NOTE_F4,DURATION_OVERRIDE_MINUS_8), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_A4,DURATION_OVERRIDE_8),
+
+      (OVERRIDE_NOTE_AS4,DURATION_OVERRIDE_MINUS_8), (OVERRIDE_NOTE_A4,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_8), (OVERRIDE_NOTE_FS4,DURATION_OVERRIDE_MINUS_8), (OVERRIDE_NOTE_E4,DURATION_OVERRIDE_16), (OVERRIDE_NOTE_FS4,DURATION_OVERRIDE_8),
+      (OVERRIDE_NOTE_G4,DURATION_OVERRIDE_MINUS_2)
+//    then it just repeats once more in the arduino code. Decided to DRY it.
```

Comment:
You can add a `flatMap` here to duplicate the array

Like this: https://github.com/danilofuchs/ForceTouchPlayer/blob/main/ForceTouchPlayer/Songs/Tetris.swift

Reply:
Yes, but as a user I feel more comfortable pressing the button again if I want to listen to exactly the same thing again